### PR TITLE
English (US) as one of the default languages and some usage information in the readme

### PR DESCRIPTION
**Pull request by IgelRM**
Hi, this is a really useful tool for me since game developers ever so often don't add languages options to their games (I used this for "The Walking Dead Michonne" btw). I was a bit surprise to find Japanese as the default language available after install since I imagine more people would use a tool like this to run software in English. So I just created this really basic pull request.

I also added some usage information in the readme so it's a bit easier to understand what which file does.

Also a button to open up the "edit global profile list tool" from the installer that pops up after successful installation would be nice for an intuitive installation, but I don't have VS installed currently to implement that.

This is my first github pull request, so I don't know how to cherry pick my commits, sorry. Exclude 	2d0cc17 and fadc9fc.

Edit: I implemented the edit global profile list button now and fixed the timezone on the English (US) language.

Edit2: Just an improvement idea: Multi-language support for the LEInstaller would be nice to implement.

### CODE REVIEW

**Comment by xupefei** [comment]
I am afraid I can not merge this request. LE is designed for running [Galgames](https://en.wikipedia.org/wiki/Bish%C5%8Djo_game) on all PCs, and this is the reason why Japanese is on the default profile list. I don't think much people use it for running English games.

Regard to one of your suggestion for adding "_... a button to open up the "edit global profile list tool" from the installer that pops up after successful installation_", I think you are right. I will implement it soon.

> Multi-language support for the LEInstaller

The installer is never used after the installation. Make it multilingual is quite a waste of time.

**Comment by IgelRM** [comment]
Hey, thanks for answer. I did not know about that this was aimed at those specific games. Changing the languages after installation is easy if one uses this for other games regardless, so I understand that you want to keep Japanese as the default language.

On LEInstaller multi-language support, you are right that people who install such a tool probably know enough English to go through a Installer that is only needed once but it's still a nice feature for those who just don't. But it's also just a lot of work with not much use, so I don't judge you.

On the button, I implemented in the end, it's 46d6425. The users still needs to press it in the end though, I wrote it in the text.

Some instructions and usage info for the readme on GitHub would be nice, I did basic ones on the fork's readme. And some sentences on global profiles and application profiles.

**Comment by xupefei** [comment]
Thanks for your commit 46d6425571b532259b339f1a53f04d238d08b025. I have left some comments for improvements there so please have a look. After that I will happy to cherry-pick that commit into the main repo.

File: LEInstaller/Form1.cs [hunk 1]
```diff
@@ -58,6 +58,7 @@ private void Form1_Load(object sender, EventArgs e)
 
             buttonUninstall.Enabled = ShellExtensionManager.IsInstalled(false);
             buttonUninstallAllUsers.Enabled = ShellExtensionManager.IsInstalled(true);
+            buttonEditGlobal.Enabled = ShellExtensionManager.IsInstalled(true);
```
Comment (xupefei) [inline LEInstaller/Form1.cs]: `IsInstalled(true)` only cares about the installation for all users in system. You should use `IsInstalled(true) || IsInstalled(false)`.

**Comment by IgelRM** [comment]
Thanks, for the details. I am not an advanced coder as you probably saw from the mistakes. I applied your improvements, 6e2c988.

**Comment by xupefei** [comment]
It's done (not vis this PR). Thanks for your effort!

**Pull request closed without merging**